ComfortZoneApp/app/src/main/res/values/colors.xml

```diff
@@ -8,5 +8,6 @@
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
     <color name="grey">#ABE5E5E8</color>
-    <color name="blue">#6175DF</color>
+    <color name="blue">#67B3E6</color>
+    <color name="light_blue">#DAF3FF</color>
```

Comment:
it's nice that you are adding colors spec here and actually using them, but we should be more descriptive of what ui is it for (i.e. profile border/background), sometimes, you want to be more generic too, but we shouldn't just name it as the color.